themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -32,7 +52,15 @@ region=Estado, Provincia, o Regi\u00F3n
 postal_code=C\u00F3digo Postal
 country=Pa\u00EDs
 emailVerified=Email verificado
-gssDelegationCredential=GSS Delegation Credential
+gssDelegationCredential=Delegaci\u00F3n de credenciales GSS
+
+profileScopeConsentText=Perfil de usuario
+emailScopeConsentText=Email
```

Comment:
In this file there are some translations of "Email" as "correo electrónico" and others as "email", but it would be better to use always "correo electrónico"